indexer/feature_loader.cpp

```diff
@@ -292,6 +292,48 @@ void LoaderCurrent::ParseMetadata()
   }
 }
 
+void LoaderCurrent::ParseAltitude()
+{
+  // @TODO Index and should be used here before merging to master.
+
+  if (m_Info.GetMWMFormat() < version::Format::v8)
+    return;
+
+  struct TAltitudeIndexEntry
+  {
+    uint32_t featureId;
+    feature::TAltitude beginAlt;
+    feature::TAltitude endAlt;
+  };
+
+  try
+  {
+    DDVector<TAltitudeIndexEntry, FilesContainerR::TReader> idx(m_Info.GetAltitudeReader());
```

Comment:
Please, don't.
I'm against the idea to incorporate m_altitudeParsed and so on into the FeatureType class. There should be individual AltitudeLoader (or something like that) with quite simple API, like:
```c++
class AltitudeLoader
{
public:
  AltitudeLoader(MwmValue & value);

  Altitudes GetAltitudes(uint32_t featureId);
};
```